### ISSUE TITLE
Update email timestamps to place time below date

### ISSUE THREAD

**Issue by jodyheavener** [mozilla/fxa#8013]
## Description

In many of our emails we include a block of information including a date and time:

![Screen Shot 2021-03-26 at 3 31 44 PM](https://user-images.githubusercontent.com/6392049/112685569-289cfb00-8e4b-11eb-8035-5a9bbd72e26e.png)

Let's move the time to it's own line, after the date line.

This will require modifying the `location` template [partial](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/lib/senders/templates/partials/location.html#L6) (as well as the [text version](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/lib/senders/templates/partials/location.txt#L5:L5)) to add a new line to render the `time`, and maybe rename `timestamp` to `date` or similar. It'll also involve updating the mailer's usage of [`_constructLocalTimeString`](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/lib/senders/email.js#L374:L381), as well as updating any place where a mailer [passes](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/lib/senders/email.js#L936:L940) in the `timestamp` variable to use our new date/time variables (whatever they're named, you decide).

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-3280)
┆Issue Number: FXA-3280

**Comment by flozender** [comment]
Hi, I'd love to take this up. May I?

**Comment by jodyheavener** [comment]
@flozender sure thing, it's all yours! You may want to check out how to set up [MailDev](https://github.com/mozilla/fxa#running-with-maildev) locally for testing, and the auth server's [`write-emails`](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/package.json#L27) command may also prove helpful for writing mail templates to disk.

**Comment by flozender** [comment]
I'm trying to set up the repo locally but I get thrown this error: 
```
start fxa services...

###########################################################

# fxa couldn't start. see ./artifacts/start.log for details

###########################################################

The script called "start.default" which runs "pm2 start _dev/pm2/infrastructure.config.js && _scripts/pm2-all.sh start && echo "Use 'npm stop' to stop all the servers"" failed with exit code 1 https://github.com/sezna/nps/blob/master/other/ERRORS_AND_WARNINGS.md#failed-with-exit-code
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! fxa@2.0.0 start: `nps --prefix=start`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the fxa@2.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/tayeeb/.npm/_logs/2021-03-30T12_09_10_327Z-debug.log
```
Here's the [./artifacts/start.log](https://paste.mozilla.org/CCDcRpzz/raw)

Can you please tell me if I'm missing something here?

**Comment by jodyheavener** [comment]
@flozender These can be tricky to debug, so I would recommend asking in the [FxA Matrix channel](https://chat.mozilla.org/#/room/#fxa:mozilla.org) so it gets more attention. From this error it looks workspaces related, so can you double check that you're using the correct [Node version](https://github.com/mozilla/fxa/blob/main/.nvmrc), that you're running your commands with `yarn` and not `npm`, and that your dependencies are fully installed across workspaces (running `yarn install` from the root directory should cover this).

**Comment by Chiamaka-Uzuegbu** [comment]
Hello Jody, Can I try this task out too?

**Comment by data-sync-user** [comment]
➤ Vasilica Tamas commented:

This issue is verified as fixed on Stage [Train 1.204.3] under Windows 10 64-bit. I confirm that the time is successfully placed under the date in FXA related emails.
